### DISCUSSION TITLE
p/fbc92423364d486c9ad7f3ebe30f14a6/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#12] [Announcements]
# p/fbc92423364d486c9ad7f3ebe30f14a6/

缘由今年春节回来有几天，家乡身处北方农村，到昨天为止吾辈已经难以忍受家里这些鸟事，今天打算就回北京了。曾经，吾辈讨厌复杂的人际关系而转向了计算机，相比于人与人之间的模糊与暧昧不清，计算机要直白的多，对就是对，错就是错，即便是老板提出了“根据手机壳改变应用程序背景颜色”这种不合理的需求也可以拒绝掉，就像 1 + 1 不可能等于 3 一样。 工作怎么样？收入多少钱？基本上，每一个人见到面的亲戚的前 3

https://blog.rxliuli.com/p/fbc92423364d486c9ad7f3ebe30f14a6/

**Comment by kahosan** [comment]
同感

**Reply by ethuh** [reply]
说的太好了，我爸爸也是爱喝酒，身体喝垮了，没有任何一个人能劝他，我记得在我读初中的时候，喝进医院，哭着说再也不喝了，但是并没有什么用。我从小就痛恨喝酒，每一个劝我喝酒的人我都莫名的反感，已经是下意识的反应了。即使现在回家看望父母，我爸依然坚持让我喝点，似乎我不和他喝点酒，就没有话可以聊，就没法开始聊。

**Reply by ethuh** [reply]
sorry，本应该是回复博主的，错误的回复到你这里了

**Comment by anoumous4725** [comment]
我40了，单身躺平族，5年没回家了，都是以上原因。天朝文化就是这样，既然我接受不了，我可以拒绝远离，我可以说不

**Reply by rxliuli** [reply]
是啊，接受不了，也无法改变，那也只能逃离了呢（

**Reply by anoumous4725** [reply]
哇，你用的这个评论系统giscus很好啊，还会实时刷新，收藏一波

**Comment by anoumous4725** [comment]
嗯说得不对，不是天朝文化就是这样，而是天朝已经没有文化（毛40年运动、文革的成果），几乎所有人眼中只有物质、权力

**Reply by XingrunLi** [reply]
太真实了o(￣┰￣*)ゞ每次过年在家就跟坐牢一样

**Reply by rxliuli** [reply]
或许以后不在春节的时候回去，而是选择平时请年假或其他假期回去
